ground/gcs/src/plugins/coreplugin/mainwindow.cpp

```diff
@@ -181,14 +181,28 @@ MainWindow::MainWindow() :
     registerDefaultContainers();
```

Comment:
All elements have border, margin and padding fields. I'd expect a mark-up based layout engine to implement a proper boxing model, that's consistent. Poking the stick at it, trying to make various visuals work on an experimental branch, there's occasionally a catch that makes you scratch your head. In the case of the central widget, I had to nest it inside another widget to make it work as expected, so that I could apply paddings to the MyTabWidget.